packages/unlock-express/example/app.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable no-unused-vars */
+const express = require('express')
+const cookieParser = require('cookie-parser')
+
+const configureUnlock = require('../index')
```

Comment:
The example should be self-contained if possible could this be moved to the current directory?